content/vendor/guides/helm-chart.md

```diff
@@ -28,83 +27,219 @@ A full example of the code for this guide can be found in the [kotsapps reposito
 
 ### Getting a Chart Archive
 
-##### With helm fetch
+We can get the Helm Chart archive in a couple of ways, either using `helm fetch` or `helm package`.
 
-- helm repo add ...
-- VALIDATE: helm repo ls 
-- helm fetch stable/grafana
-- VALIDATE: tar xOf grafana-5.0.11.tgz grafana/Chart.yaml
+##### Using helm fetch
 
-##### With helm package
+Add a Helm repo
 
-- git checkout https://github.com/helm/charts
-- cd charts/stable/grafana
-- you can use the latest, but we'll be working off of this commit: `git checkout f2fcadaa5` (just to be sure)
-- helm package .
-- VALIDATE: tar xOf grafana-5.0.11.tgz grafana/Chart.yaml, you should see
+```shell
+helm repo add stable https://kubernetes-charts.storage.googleapis.com
```

Comment:
when showing single-line shell commands with their output, I tend to try to put a `$` at the start of the line for the commands, just to visually differentiate between the command and the output

```suggestion
$ helm repo add stable https://kubernetes-charts.storage.googleapis.com
```